deps/rabbitmq_mqtt/test/reader_SUITE.erl

```diff
@@ -299,14 +299,13 @@ rabbit_mqtt_qos0_queue_overflow(Config) ->
     %% We expect that
     %% 1. all sent messages were either received or dropped
     ?assertEqual(NumMsgs, NumReceived + NumDropped),
-    case rabbit_ct_helpers:is_mixed_versions(Config) of
-        false ->
+    case rabbit_ct_broker_helpers:is_feature_flag_enabled(Config, rabbit_mqtt_qos0_queue) of
```

Comment:
You should try to enable the feature flag, then handle the error if any and fallback to an alternative code path.

It's possible that both the old and new versions support the feature flag. It's just that in mixed-version mode, all non-required feature flags are disabled by default when nodes start.

Reply:
Thanks. For this specific test case, I pushed a commit where this feature flag gets always enabled because this test case doesn't make sense to be executed without `mqtt_qos0` queue type being supported.